packages/rs-drive-abci/tests/strategy_tests/strategy.rs

```diff
@@ -1921,15 +1993,200 @@ impl NetworkStrategy {
         )
     }
 
-    fn create_identity_top_up_from_addresses_transitions(
+    fn create_identity_top_up_from_addresses_transitions<S: Signer<PlatformAddress>>(
```

Comment:
If I understand correctly, you moved away from my test scenario (address top up from core + send from address to identity in the same block), so we don't cover that case ?